csv_to_entities.py

```diff
@@ -0,0 +1,97 @@
+"""
+python csv_to_entites.py
+
+accept in a UTF8 csv file and a delimiter and converts it to entities
+your csv should be of the format
+no headers
+filename without .csv is entity name
+col0 = key_value
+col1 = onward are synonyms
+for example this is the way Dialogflow exports an entity
```

Comment:
Checked both CX and ES. Only in ES you could download entities as CSV whereas in CX you cannot download entities separately. It needs to be downloaded as entire agent zip file and in that we would find JSON.